mcp/protocol.go

```diff
@@ -893,7 +893,30 @@ type ResourceUpdatedNotificationParams struct {
 
 // TODO(jba): add CompleteRequest and related types.
 
-// TODO(jba): add ElicitRequest and related types.
+type ElicitParams struct {
+	// This property is reserved by the protocol to allow clients and servers to
+	// attach additional metadata to their responses.
+	Meta `json:"_meta,omitempty"`
+	// A human-readable message describing what information is being requested.
+	Message string `json:"message"`
+	// A JSON Schema object defining the structure of the requested data.
+	RequestedSchema *jsonschema.Schema `json:"requestedSchema"`
+}
+
+func (x *ElicitParams) GetProgressToken() any  { return getProgressToken(x) }
+func (x *ElicitParams) SetProgressToken(t any) { setProgressToken(x, t) }
+
+// The client's response to an elicitation/create request from the server.
+type ElicitResult struct {
+	// This property is reserved by the protocol to allow clients and servers to
+	// attach additional metadata to their responses.
+	Meta `json:"_meta,omitempty"`
+	// The action taken by the user: "accept", "decline", or "cancel".
+	Action string `json:"action"`
+	// The data provided by the user if action is "accept". This should conform
```

Comment:
Here too. Use the description from the Schema file

Reply:
done